docs/server/settings.rst

```diff
@@ -294,20 +298,50 @@ Translation environment configuration settings.
             'HOST': 'localhost',
             'PORT': 9200,
             'INDEX_NAME': 'translations',
+            'TM_NAME': 'Local translations',
+            'WEIGHT': 1,
+            'IS_LOCAL': True,
+            'MIN_SCORE': 'AUTO',
+        },
+        'external': {
+            'ENGINE': 'pootle.core.search.backends.ElasticSearchBackend',
+            'HOST': 'localhost',
+            'PORT': 9200,
+            'INDEX_NAME': 'external-translations',
+            'TM_NAME': 'External translations',
+            'WEIGHT': 0.9,
+            'IS_LOCAL': False,
             'MIN_SCORE': 'AUTO',
         },
     }
 
+
   This is configured to access a standard Elasticsearch setup.  Change the
   settings for any non-standard setup.  Change ``HOST`` and ``PORT`` settings
   as required.
 
   Use ``MIN_SCORE`` to set the Levenshtein Distance score.  Set it to ``AUTO``
-  so that Eslasticsearch will adjust the required score depending on the length
+  so that Elasticsearch will adjust the required score depending on the length
   of the string being translated. Elasticsearch documentation provides further
   details on `Fuzzy matching
   <https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#fuzziness>`_.
 
+  ``IS_LOCAL`` tells whether the TM is a local TM or an external TM. Local TMs
+  are populated using translations done within Pootle, and they are
+  automatically updated every time a new translation is sent. External TMs are
+  populated using translations coming from files holding translations, such as
+  regular translation files or translation memories (TMX files). External TMs
+  are thus not automatically updated.
```

Comment:
I'm OK with ``AUTO_UPDATE`` but I'm also OK to use the name ``local`` instead of ``default`` to signal.  Though I think I prefer the first as there isn't any inferred behaviour.